lib/api/apiUtils/coldStorage.js

```diff
@@ -0,0 +1,33 @@
+/*
+ * Code based on Yutaka Oishi (Fujifilm) contributions
+ * Date: 11 Sep 2020
+ */
+
+/**
+ * Get response header "x-amz-restore"
+ * Be called by objectHead.js
+ * @param {object} objMD - object's metadata
+ * @returns {string} x-amz-restore
+ */
+function getAmzRestoreResHeader(objMD) {
+    let value;
```

Comment:
This var isn't required and I would just return instead of setting it. Use `if` instead of `else if` as well

Reply:
I now return the value, or undefined: https://github.com/scality/cloudserver/pull/4504/commits/a30ae19648cf97efb17ed42ef27c84bbf91b3fff#diff-5c1a41d999742cdc5b48a81be655c072599261c1755387ef030e91f2f54105b6R19